app/config/services.yml

```diff
@@ -124,7 +124,16 @@ services:
         autowire: true
         tags:
             - { name: monolog.logger, channel: login_attempts }
- 
+
+    pelagos.login_listener:
+        class: Pelagos\Event\LoginEventListener
+        arguments: ["@security.token_storage"]
+        tags:
+          - { name: kernel.event_listener, event: security.interactive_login, method: onSecurityInteractiveLogin }
```

Comment:
Instead of using this new event, you can just stick your code in LoginFormAuthenticator. It has a nice function for you:
`public function onAuthenticationSuccess(Request $request, TokenInterface $token, $providerKey)
    {
        dump('HELLO WORLD!');
    }`